spec/models/lettings_log_spec.rb

```diff
@@ -1514,6 +1514,24 @@ def previous_postcode_fields
       expect(record_from_db["has_benefits"]).to eq(1)
     end
 
+    describe "deriving vacant days" do
```

Comment:
These should be written in a way that makes it easy to solve merge fun with https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2442 - but will need to do that because there wasn't a way to avoid entirely.